code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -32,7 +32,7 @@
 	environment_smash = 0
 	mouse_opacity = 2
 	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
-	mob_size = MOB_SIZE_SMALL
+	mob_size = MOB_SIZE_TINY
```

Comment:
@Incoming5643 no they're still hard as balls to click without mouse opacity, they move super fast and their sprite jitters up and down